app/assets/stylesheets/identifications/by_login.scss

```diff
@@ -0,0 +1,46 @@
+@import "../colors";
+
+.observations.mini .observation {
+  padding: 10px 0;
+  margin-bottom: 0;
+  border: 0;
+}
+
+.identification {
+  padding: 10px;
+  background-color: #eee;
+
+  .identification_image { width: 75px; }
+  .identification_body { float: left; }
```

Comment:
Sorry I missed this on the first pass: remove this and add `display: flex` to `.identification` to achieve the same effect while not wrapping at smaller window sizes.

Reply:
Fixed, thanks. 
I put `display: flex` under the bootstrap section, as the bootstrap_bundle css of `display: block` seemed to be overriding it.